apisix/plugins/limit-count.lua

```diff
@@ -188,7 +188,7 @@ function _M.access(conf, ctx)
         if conf.allow_degradation then
             return
         end
-        return 500, {error_msg = "failed to limit count: " .. err}
```

Comment:
Let's fix the message above: `core.log.error("failed to limit req: ", err)`

Reply:
okay, thanks~ done~